Cargo.toml

```diff
@@ -9,58 +9,56 @@ edition = "2021"
 include = ["src/**/*", "LICENSE-*", "*.md"]
 
 [dependencies]
-clap = { version = "~4.5", features = ["derive", "std", "help"], default-features = false }
-git2 = { version = "~0.19", features = ["ssh", "https"], default-features = false }
+anstyle = "1.0.7"
+anyhow = "~1.0"
 auth-git2 = "~0.5"
+clap = { version = "~4.5", default-features = true, features = [
+    "derive",
+    "std",
+    "help",
+    "wrap_help",
+] }
```

Comment:
This is the only real change in this file (adding wrap help and turning on default features, which activates the color and the usage features)